scootapi/client/cli.go

```diff
@@ -49,28 +49,31 @@ func NewSimpleCLIClient(d dialer.Dialer) (CLIClient, error) {
 }
 
 func (c *simpleCLIClient) Dial() error {
-	_, err := c.dial()
-	return err
+	// Always recreate a connection.  This way if something went wrong with
+	// the previous connection we don't have a clean one.
```

Comment:
Yes.  The goal is to always have a clean connection.  Updated the comment to be a little less rambly :)